test/input_gen/gen_layer_tests.py

```diff
@@ -171,6 +171,21 @@ def call(self, inputs):
     record_single(conv, (1, 3, 11, 11), "conv2d_sb_same_dilation")
     record_single(conv, (3, 3, 11, 11), "conv2d_mb_same_dilation")
 
+    conv = K.layers.UpSampling2D(size=(2, 2), interpolation="nearest", input_shape=(2, 2, 1))
```

Comment:
I think you need to run this script on your local environment to create a golden data file and then compress that golden files to the `unittest_layers.tar.gz` in the nntrainer/packaging folder.